mathesar/tests/api/query/test_library_demo_reports.py

```diff
@@ -93,13 +93,19 @@ def check_overdue_books_columns(create_overdue_books_query, client):
             'display_name': 'Titles',
             'type': '_array',
             'type_options': {'item_type': 'text'},
-            'display_options': None
+            'display_options': None,
+            'is_initial_column': False,
+            'is_summarization_output_column': True,
+            'summarization_input_alias': 'Book Title',
         }, {
             'alias': 'email',
             'display_name': 'Patron Email',
             'type': 'mathesar_types.email',
             'type_options': None,
-            'display_options': None
+            'display_options': None,
+            'is_initial_column': True,
+            'base_table_name': 'Patrons',
+            'base_column_name': 'Email',
```

Comment:
@dmos62 Can we change the names here to `input_table_name` and `input_column_name` so we don't confuse this terminology with the `base_table` we already have?